StackExchange/Stacks
# chore(sidebarwidgets): refresh styles

[STACKS-673](https://stackoverflow.atlassian.net/browse/STACKS-673)

This PR refreshes styles for the Stacks Sidebar Widget component, per the [Figma](https://www.figma.com/design/CGU7JFDfnKDtX8QXCm1I2f/Sidebar-Widgets?node-id=1-2&node-type=canvas&t=10krLqtDcZCwjKmG-0).

## How to test

> [!TIP]
> ~~Netlify deploy previews are not working right now, so you'll need to run this locally to see the changes~~ 
> **[Dev preview](https://6723abd5b8988b439233e329--stacks.netlify.app/)**

1. Visit the **[Dev preview](https://6723abd5b8988b439233e329--stacks.netlify.app/product/components/sidebar-widgets/)**
    - To test locally, checkout out this branch (`STACKS-673/sidebarwidget-style-refresh`) locally and run it (`npm i && npm run start`), head to [the sidebar widgets docs page](http://localhost:8080/product/components/sidebar-widgets/) to see the changes

## Screenshots

### Basic

<details>
<summary>Before</summary>

![image](https://github.com/user-attachments/assets/d6f85660-e376-4f03-938b-f9e131a28473)

</details>

<details>
<summary>After</summary>

![image](https://github.com/user-attachments/assets/3dc6d311-7fbc-441f-b0a8-532e8a2492cb)

</details>


### Navigation

<details>
<summary>Before</summary>

![image](https://github.com/user-attachments/assets/9714c7da-3a57-46b6-84d2-9c4b3f90ed67)

</details>

<details>
<summary>After</summary>

![image](https://github.com/user-attachments/assets/046839ae-5dcb-4adc-95d0-2e2d4c39e57d)

</details>

### Header actions

<details>
<summary>Before</summary>

![image](https://github.com/user-attachments/assets/27007e52-71ed-49e9-8101-2df0e93e52ae)

</details>

<details>
<summary>After</summary>

![image](https://github.com/user-attachments/assets/9ac43c99-e104-4dd9-9bd6-bfa5b03d6f08)

</details>

### Accordion

<details>
<summary>Before</summary>

![image](https://github.com/user-attachments/assets/26d5ceda-a7ab-4290-a564-20e6e8cac74d)

</details>

<details>
<summary>After</summary>

![image](https://github.com/user-attachments/assets/d81a0089-9a24-4f7d-a8c0-64cbaf045b39)

</details>
